newIDE/app/src/VariablesList/VariablesEditorDialog.js

```diff
@@ -11,12 +11,26 @@ import useDismissableTutorialMessage from '../Hints/useDismissableTutorialMessag
 import { Column, Line } from '../UI/Grid';
 import VariablesList from './VariablesList';
 import HelpButton from '../UI/HelpButton';
+import { getVariableContextFromNodeId } from './VariableToTreeNodeHandling';
 
 const gd: libGDevelop = global.gd;
 
+const getVariablePathFromNodeId = (
+  nodeId: string,
+  variablesContainer: gdVariablesContainer
+): string => {
+  const variableContext = getVariableContextFromNodeId(
+    nodeId,
+    variablesContainer
+  );
+  const variablePath = variableContext.lineage.map(variable => variable.name);
+  variablePath.push(variableContext.name);
+  return variablePath.join('.');
```

Comment:
Will this work with arrays?

Reply:
Yes, for array children, it does something like `MyArray.3`. It's probably not what users want if they added an array and have one child selected, but I guess that's ok.

Do you think we should remove the child for these cases?
I guess, at some point, the field could choose according to the type of variable that is needed.